recipes/doublemetaphone/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "DoubleMetaphone" %}
+{% set version = "0.1" %}
+{% set md5 = "86e5ae4a4839321f60b9801837eb01c1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win and linux]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  # Python imports
+  imports:
+    - doublemetaphone
+
+  # commands:
+    # You can put test commands to be run here.  Use this to test that the
+    # entry points work.
+
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  license: MIT License
```

Comment:
Please open an issue upstream asking for a license file and its inclusion in `MANIFEST.in`. Also please add a comment here to include it.